dask/bag/random.py

```diff
@@ -0,0 +1,123 @@
+import random
+
+
+def sample(population, k):
+    """
+    Chooses k unique random elements from a population bag.
+
+    Returns a new bag containing elements from the population while
+    leaving the original population unchanged.
+
+    Parameters
+    ----------
+    population : Bag
+        Elements to sample.
+    k : integer, optional
+        Number of elements to sample.
+    Examples
+    --------
+    >>> import dask.bag as db
+    >>> from dask.bag import random
+    >>>
+    >>> b = db.from_sequence(range(5), npartitions=2)
+    >>> len(list(random.sample(b, 3).compute()))
```

Comment:
I understand that you are showing len here because it'll always be 3, but it'd be more useful to show an actual response and just add the skip docktest flag.

Reply:
uhh thanks! :-D